src/core/datatypes/trees/TopologyNode.cpp

```diff
@@ -1858,14 +1861,7 @@ void TopologyNode::setAge(double a, bool propagate)
     // we also need to recompute the branch lengths of my children
     for (auto& child: children)
     {
-        if ( child->isTipSampledAncestor() )
```

Comment:
This doesn't seem like the right fix.  The internal-node-sampled-ancestor is supposed to have the same time as the tip-sampled-ancestor.  So, the real problem is that that something is trying to change the time of the internal-node-sampled-ancestor.  Why is this happening?

Reply:
Because we have our standard move, the mvNodeTimeSlideUniform that changes internal node ages. We could change the move, but I think that would not be safe. It would require knowledge for all moves that they are only allowed to change internal nodes if those nodes are not "fake" because of sampled ancestors. That would mean new developers of node time change moves would need to know about this behavior even though the developer might never work with fossils.

Reply:
So, what happens when mvNodeTimeSlideUniform changes the internal node age for a sampled ancestor to be different than the corresponding tip node for the sampled ancestor?